### ISSUE TITLE
New release planned ?

### ISSUE THREAD

**Issue by eduard-passnfly** [devartis/passbook#51]
@fara @poligarcia @mbaechtold 

Good morning, are you planning to do a release with all the new changes (specially support for Python 3)?

Current release dates from 25 Jul 2016 (v1.0.2). It would be great to get the latest code when installing the packages via `pip`.

thanks!

**Comment by mbaechtold** [comment]
Hey @eduard-passnfly 

While waiting for a new release to be published on PyPI, you could tell `pip` to install this Python package directly from VCS, see https://pip.pypa.io/en/stable/reference/pip_install/#vcs-support

By providing an exact commit hash, you're even able to achieve a repeatable installation of the package, e.g. `pip install git+https://github.com/devartis/passbook.git@cdd500d67e8dd007d83133cbd890e4fdd5b038e5#egg=passbook` where https://github.com/devartis/passbook/commit/cdd500d67e8dd007d83133cbd890e4fdd5b038e5 is the latest commit as of right now.

**Comment by eduard-passnfly** [comment]
Hi @mbaechtold 

Thank you for the information, this should the trick for now, I will give it a try!

Is there a way to specify a commit hash in a `requirements.txt` file ?

**Comment by mbaechtold** [comment]
Yes, in editable mode: https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format.

Putting the following line in your `requirements.txt` should do the trick (including the `-e` in front of the URL):

```
-e git+https://github.com/devartis/passbook.git@cdd500d67e8dd007d83133cbd890e4fdd5b038e5#egg=passbook
```

**Comment by eduard-passnfly** [comment]
Thank you @mbaechtold , that is useful help, I appreciate it!

**Comment by shivaRamdeen** [comment]
I created a fork that uses cryptography library instead of m2crypto if anyone is looking for an alternative, especially for windows dev env.

[https://github.com/shivaRamdeen/passbook](Here is is)

**Comment by mbaechtold** [comment]
@shivaRamdeen, it would be great if you could create a pull request with your modifications. I'd appreciate switching to `cryptography`. What do you think?

**Comment by wrvdklooster** [comment]
Thanks for the library. Will there be a new release somewhere soon? Now using specific commit hash but referring to a version would be nice.

**Comment by mbaechtold** [comment]
It would be a good idea to tag the next release as `2.0.0` since it dropped support for Python 2 and is thus backwards incompatible.

**Comment by philipptrenz** [comment]
A `2.0.0` release using `cryptography` would be awesome, `m2crypto` causes big headache on macOS

**Comment by shivaRamdeen** [comment]
> @shivaRamdeen, it would be great if you could create a pull request with your modifications from your fork. I'd appreciate switching to `cryptography`. What do you think?

Hey @mbaechtold I have not paid much attention to this repo since I made the fork. I'm happy to make the PR. will review for any compatibility issues and create the PR

**Comment by mbaechtold** [comment]
That would be awesome. I have created an issue for this: https://github.com/devartis/passbook/issues/59
